src/commands/branches.test.ts

```diff
@@ -166,6 +166,40 @@ describe('branches', () => {
     },
   });
 
+  testCliCommand({
+    name: 'create with fixed size CU',
+    args: [
+      'branches',
+      'create',
+      '--project-id',
+      'test',
+      '--name',
+      'test_branch_with_fixed_cu',
+      '--cu',
+      '2',
+    ],
+    expected: {
+      snapshot: true,
+    },
+  });
+
+  testCliCommand({
```

Comment:
There's not way to run this test with "only" (also no way to run single file from CLI).
I hope we can change that.

Reply:
I agree, lets decouple whatever is causing this

Reply:
I guess we'll first move to `vitest` and then I can make a refactor (see discussion on mocks for example).

Reply:
(#235 is mostly ready for review)

Reply:
Oh i didn't get your original question (or was looking at a different one when answering...) 
You can run a single file tests.
`npm test -- --watch`. and there you would have option to filter tests to execute